README.md

```diff
@@ -2,7 +2,9 @@
 
 > A peer-to-peer hypermedia protocol to make the web faster, safer, and more open.
 
-[![ipfs](https://img.shields.io/badge/irc-%23ipfs-brightgreen.svg)](https://webchat.freenode.net/?channels=ipfs)
+[![IRC](https://img.shields.io/badge/irc-%23ipfs-brightgreen.svg?style=flat-square)](https://webchat.freenode.net/?channels=ipfs)
+[![Matrix](https://img.shields.io/badge/matrix-%23ipfs%3Apermaweb.io-blue.svg?style=flat-square)](https://riot.permaweb.io/#/room/#ipfs:permaweb.io)
```

Comment:
* the official room is #ipfs:matrix.org.
* let's put matrix first at this point (matrix, IRC, discord).

Reply:
#ipfs:permaweb.org leads to the same room as the matrix.org alias, i just used permaweb.io since that doesn't depend on matrix.org to work.
Will move matrix first

Reply:
I know, but linking to permaweb indicates that it's somehow the preferred/default address.

Reply:
fair enough, i'll change it to matrix.org then